### DISCUSSION TITLE
Transform Landmark Support

### DISCUSSION THREAD

**Discussion by lukas-folle-snkeos** [Project-MONAI/MONAI#6509] [Ideas]
It would be great if the `monai.transforms.spatial` transformations would also be able to operate on 2D or 3D points. 

For example:
The `Resize` transformation could do the following:
image [100, 100] -> image [10, 10]
landmark (20, 40) -> landmark (2, 4)

Is something like this planed?

**Comment by wyli** [comment]
thanks for the suggestion, some coordinate operations are implemented as part of the bounding box transforms (in `monai.apps`) for detection tasks https://github.com/Project-MONAI/MONAI/blob/e5cf74aac60cb3512d92f9abb3dd48b5ed36e8d0/monai/apps/detection/transforms/array.py#L267-L270 enhancing/integrating these into the core transform modules doesn't currently have a high priority...if we can identify some critical use cases, we'll have more developments in these modules.

**Comment by lukas-folle-snkeos** [comment]
Thanks for the hint!
